react-validated-input-component/my-app/src/ValidatedInput.js

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+import './ValidatedInput.css';
+
+export default function ValidatedInput() {
+  const [password, setPassword] = useState('');
+
+  let message = '';
+  let icon = 'fa-x';
+
+  if (password === '') {
+    message = 'A password is required.';
+    icon = 'fa-x';
+  } else if (password.length < 8) {
+    message = 'Your password is too short.';
+    icon = 'fa-x';
+  } else {
+    icon = 'fa-check';
+  }
+
+  return (
+    <form className="form-group">
+      <label htmlFor="password5">Password</label>
```

Comment:
If you wrap this `label` around the `input` you don't need `htmlFor` or the input's `id`.